File: 1. Model/Produto.py


class Produto:

    def __init__(self, nome, tipo, desc):
        self.nome      = nome
        self.tipo      = tipo
        self.descricao = desc





    #  - - - - - - - - - - GETS & SETS - - - - - - - - - -
    ##GETS
    def getNome(self):
        return self.nome

    def getTipo(self):
        return self.tipo

    def getDescricao(self):
        return self.descricao


    ##SETS
    def setNome(self, n):
        self.nome = n

    def setTipo(self, t):
        self.tipo = t

    def setDescricao(self, d):
        self.descricao = d
    # - - - - - - - - - - //// /  //// - - - - - - - - - -

File: model/Produto.java
package model;



public class Produto {

	private String nome;
	private String tipo;
	private String descricao;



	public Produto(String nome, String tipo, String descricao){

		this.nome 	   = nome;
		this.tipo 	   = tipo;
		this.descricao = descricao;
	}




	// Gets
	public String getNome() {
		return this.nome;
	}

	public String getTipo() {
		return this.tipo;
	}

	public String getDescricao() {
		return this.descricao;
	}





	// Sets
	public void setNome(String n) {
		this.nome = n;
	}

	public void setTipo(String t) {
		this.tipo = t;
	}

	public void setDescricao(String d) {
		this.descricao = d;
	}
}
